test/lib/mayaUsd/fileio/testEditAsMayaBBox.py

```diff
@@ -66,29 +66,32 @@ def testGroupBoundingBox(self):
             self.assertTrue(mayaUsd.lib.PrimUpdaterManager.canEditAsMaya(sphere2UfePathStr))
             self.assertTrue(mayaUsd.lib.PrimUpdaterManager.editAsMaya(sphere2UfePathStr))
 
+        cmds.select('Sphere2', r=True)
+        cmds.move(10., 10., 10., relative=True)
```

Comment:
Please add a comment here about why you are moving the Maya object (to make sure that after the edit as Maya, the Maya object is not the same as the Usd prim. Then test the BBox - it should be different.